### PR TITLE
Dev/header

### PR DESCRIPTION
**Pull request by arjund-06**
### Functionality:
Makes Navbar more funtional.

### Solution:
Grouped the create task and start sprint buttons and grouped the board buttons and added button to create team and add members. Added focus nav to the board component.

### Risk level:
- [ ] high 
- [ ] medium
- [X] low

### How to test:
1. Use ng serve and see the difference in the Navbar.

Picture of navbar:

image

### CODE REVIEW

**Review by bighnesh0404 — APPROVED** [review]
working properly without any error, ship it

File: src/app/body/board/focus-nav/focus-nav.component.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+.menu-option:hover {
+    background-color: #5A70F0;
+    cursor: pointer;
+}
+
+
+/* .row {
+    width: 100%;
+} */
```
Comment (viv9k) [inline src/app/body/board/focus-nav/focus-nav.component.css:16]: Please remove the commented code

**Review by viv9k — CHANGES_REQUESTED** [review]
Please complete the required changes

**Review by viv9k — APPROVED** [review]
Ship it!!

**Pull request merged**